source/plugins/base/queries/field.repositories.graphql

```diff
@@ -1,6 +1,6 @@
-query BaseFieldRepositories {
+query BaseFieldRepositories{
   $account(login: "$login") {
-    repositories(last: 0) {
+    repositories(last: 0, $affiliations) {
```

Comment:
```suggestion
    repositories(last: 0 $affiliations) {
```
The `affiliations` already contains a comma (`,`) so no need to add one in the query (this is to support empty affiliations)

(same for other occurences)

Reply:
done 😁